.github/workflows/update-copyright-notices.yml

```diff
@@ -0,0 +1,39 @@
+name: Update copyright notices
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  check-and-update-copyright-notices:
+    if: ${{ github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./tools/update-copyright-notices.sh
+      - run: |
+          git config --global user.email "actions@github.com"
```

Comment:
Does this need to be set explicitly?

Reply:
Yes. The defaults error out and passing `git commit --author` is not sufficient either.